guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,50 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+You can select to resolve package dependencies per Content View or have it enabled by default in *Settings > Content > "content view Dependency Solving Default"*.
```

Comment:
I would actually move line 42 up here.  Something along the lines of 

> This is relevant only in limited contexts.  In the most common use cases, such as adding a security errata to a content view with incremental update, dependency solving is likely unnecessary and can cause significant delays to Content View publication.

I think the steps to enable or disable it might make sense to be the very last thing discussed, only after the description of what it does, what is the purpose and what are the downsides.

I think line 10 and line 52 are also slightly duplicated.